packages/account/tests/SmartAccountV2.local.spec.ts

```diff
@@ -337,6 +337,32 @@ describe("BiconomySmartAccountV2 API Specs", () => {
     // ((await expect(entryPoint.handleOps([signedUserOp], beneficiary))) as any).to.emit(recipient, "Sender");
   }, 10000); // on github runner it takes more time than 5000ms
 
+  it("Creates another replicated instance using void signer", async () => {
+
+    let newmodule = await ECDSAOwnershipValidationModule.create({
+      signer: new VoidSigner(await owner.getAddress()),
```

Comment:
Can add another test if it is able to generate the userOp

and then we create another instance of accountApi with real Signer and send transactions with it

Reply:
will add this in test coverage improvement PR